ajax/libs/socket.io/package.json

```diff
@@ -12,10 +12,9 @@
   "npmName": "socket.io-client",
```

Comment:
@pvnr0082t 
Uh...I think [this](https://www.npmjs.com/package/socket.io) is the right npmName.
How do you think?